spec/controllers/stories_controller_spec.rb

```diff
@@ -146,4 +146,69 @@
       end
     end
   end
+
+  describe '#UPDATE' do
```

Comment:
It's convention to match case of the method name. Solid tests, you got everything here.

Reply:
Done.